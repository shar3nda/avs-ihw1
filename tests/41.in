14
4 10 6 4 -1 5 10 1 6 3 5 3 3 1