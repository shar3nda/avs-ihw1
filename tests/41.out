-1 5 10 1 6 3 5 3 3 1